### PR TITLE
feat: add helper label for geoloc

### LINKED ISSUE #876: secondary address for geolocation

### PR DESCRIPTION
**Pull request by pyphilia**
close #876

### CODE REVIEW

File: src/services/item/plugins/geolocation/repository.ts [hunk 1]
```diff
@@ -29,12 +29,15 @@ export class ItemGeolocationRepository {
    */
   async copy(original: Item, copy: Item): Promise<void> {
     const geoloc = await this.getByItem(original);
+    console.log(geoloc);
```
Comment (spaenleh) [inline src/services/item/plugins/geolocation/repository.ts:32]: remove ?

**Review by spaenleh — APPROVED** [review]
Looks good, what can be in this helper label ?

**Comment by pyphilia** [comment]
Some things like "red door, next to the cemetary"

**Comment by pyphilia** [comment]
I will wait next week to test on dev the changes, since it has a migration.

**Pull request merged**